dynamic-test-driver/app/index.js

```diff
@@ -1,16 +1,101 @@
 "use strict";
 
 const API = require("./src/ups-api");
-const TestRunner = require("./src/test-runner");
+const AppsTestRunner = require("./src/apps-test-runner");
+const AliasTestRunner = require("./src/alias-test-runner");
+const argv = require("yargs");
+const Utils = require("./src/utils");
+const async = require("async");
 
-// Take the necessary command line arguments.
-const username = process.argv[2];
-const password = process.argv[3];
+const DEFAULT_DELAY = 1000;
+const DEFAULT_INSTANCES = 1;
+const DEFAULT_ENDPOINT = "http://localhost:8080/ag-push";
 
-if (!username || !password) {
-    console.log("usage: node index.js <username> <password>");
-    process.exit(1);
+const args = argv
```

Comment:
I think it would be better to put this into its own function (ie `const args = buildCommandLineParser();`)